src/payment/strategies/apple-pay/apple-pay-session-factory.ts

```diff
@@ -0,0 +1,5 @@
+export default class ApplePaySessionFactory {
+    create(request: ApplePayJS.ApplePayPaymentRequest): ApplePaySession {
+        return new ApplePaySession(1, request);
```

Comment:
Need to investigate if it's okay to upgrade to the latest version, which is believe is 12?

Reply:
I will look into it and see what the pros/cons are. Went with 1 since that's what we support in the cart page as of now.

Reply:
It looks like they have different OS requirements: https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_on_the_web_version_history. So v1 would be the most supported but has the least features?

Reply:
Yes that's the correct assumption. Each new version is tailored for the features that come with that particular macOS. So may have more features but less support. I think we prefer to support more macOs's?